utils/units.js

```diff
@@ -74,7 +74,15 @@ function formatUnits(value, unitType, options) {
 
 function parseUnits(value, unitType) {
     var unitInfo = getUnitInfo(unitType || 18);
-    if (!unitInfo) { throwError('invalid unitType', { unitType: unitType }); }
+
+    if (!unitInfo) {
+     if (typeof(unitType) === 'string') {
+      throwError('invalid unitType', { unitType: unitType });
+     } else {
+      var wei = bigNumberify(value).mul(Math.pow(10, parseInt(unitType)))
```

Comment:
This isn't safe to do on IEEE 754 floating point numbers, since 10 ** 18 is outside the range of safe integers.

Reply:
Okay! I'll make that modification for no greater then later once we clarify .travis.yml format.

Reply:
I've just updated this, actually. Want to check out the code changes?

https://github.com/ethers-io/ethers.js/commit/12b68b0242f5f943fe99d5b2a7a0054f362bcc27